packages/composables/src/useBilling/index.ts

```diff
@@ -15,6 +19,8 @@ const params: UseBillingParams<BillingAddress, AddParams> = {
 
     const cart = await context.$propeller.api.cart(existingCartId);
 
+    serverErrors.value.push(...cart.errors || []);
```

Comment:
if `...cart.errors` is not defined, `serverErrors.value` will have a value of `[[]]`... does that makes sense?
bet leaving it empty makes more sense